grunt/tasks/schema-defaults.js

```diff
@@ -21,14 +22,15 @@ module.exports = function(grunt) {
             var pluginCategory = pluginCategories[pluginTypeIndex];
 
             //iterate through plugins in plugin type folder
-            grunt.file.expand({filter: 'isDirectory'}, grunt.config('sourcedir') + pluginType + '/*').forEach(function(path) {
-                var currentPluginPath = path;
+            var pluginTypeGlob = path.join(grunt.config('sourcedir'), pluginType, '/*');
+            grunt.file.expand({filter: 'isDirectory'}, pluginTypeGlob ).forEach(function(pluginPath) {
+                var currentPluginPath = pluginPath;
 
                 // if specific plugin has been specified with grunt.option, don't carry on
-                if (!Helpers.isPathIncluded(path + "/")) return;
+                if (!Helpers.isPathIncluded(path.join(pluginPath, '/'))) return;
 
-                var currentSchemaPath = currentPluginPath + "/" + "properties.schema";
-                var currentBowerPath = currentPluginPath + "/" + "bower.json";
+                var currentSchemaPath = path.join(currentPluginPath, '/properties.schema');
```

Comment:
Would you mind removing the slashes for brevity?

Reply:
of course